src/hooks/useEagerlyConnect.ts

```diff
@@ -27,11 +25,6 @@ export default function useEagerlyConnect() {
 
     if (selectedWallet) {
       connect(getConnection(selectedWallet).connector)
-    } else if (!selectedWalletBackfilled) {
-      BACKFILLABLE_WALLETS.map(getConnection)
-        .map((connection) => connection.connector)
-        .forEach(connect)
-    }
-    // The dependency list is empty so this is only run once on mount
+    } // The dependency list is empty so this is only run once on mount
```

Comment:
If eslint allows it, this comment should be on the next line

Reply:
I can't because that's where i do the disable-line